app/uk/gov/hmrc/tariffclassificationfrontend/controllers/StatusChangeAction.scala

```diff
@@ -35,29 +35,29 @@ trait StatusChangeAction[T] extends RenderCaseAction {
 
   protected def status(c: Case): T
 
-  protected def chooseStatusView(c: Case, preFilledForm: Form[T])(implicit request: Request[_]): Html
+  protected def chooseStatusView(c: Case, preFilledForm: Form[T], options: Option[String] = None)(implicit request: Request[_]): Html
 
   protected def update(c: Case, status: T, operator: Operator)(implicit hc: HeaderCarrier): Future[Case]
 
   protected def onSuccessRedirect(reference: String): Call
 
   protected val requiredPermission: Permission
 
-  def chooseStatus(reference: String): Action[AnyContent] = (verify.authenticated andThen verify.casePermissions(reference) andThen
+  def chooseStatus(reference: String, options: Option[String] = None): Action[AnyContent] = (verify.authenticated andThen verify.casePermissions(reference) andThen
```

Comment:
What're the options for here? They don't seem to be used. Unlike for the chooseStatusView method, all classes that use this trait override this method, and none of them use the options. Maybe an idea that you started off with and ended up not using?

Reply:
ExtendedUseCase controller appears to use the trait method.   I wanted it in the SampleController so that I could use it to determine what form to render.

Reply:
Yep, you're right, sorry, didn't notice that the extendedusecase class fell back to the default method in trait